### PR TITLE
Fix TagRootVolume script in sc-ec2-linux-jumpcloud

### PR DESCRIPTION
**Pull request by tthyer**
fix several errors in bash command in TagRootVolume

### CODE REVIEW

File: ec2/sc-ec2-linux-jumpcloud.yaml [hunk 1]
```diff
@@ -386,14 +386,15 @@ Resources:
...
+                - - "set -x;"
+                  - "EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) &&"
                   - "ROOT_DISK_ID=$(/usr/bin/aws ec2 describe-volumes --region $AWS_REGION "
                   - "--filters Name=attachment.instance-id,Values=$EC2_INSTANCE_ID "
-                  - "--query Volumes[].VolumeId --out text); "
-                  - "EC2_INSTANCE_TAGS=$(aws --region $AWS_REGION ec2 describe-tags --filters 'Name=resource-id,Values=$EC2_INSTANCE_ID') "
-                  - "PROJECT=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"Project\").Value ') "
-                  - "DEPARTMENT=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"Department\").Value ') "
-                  - "OWNER_EMAIL=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"OwnerEmail\").Value ') "
+                  - "--query Volumes[].VolumeId --out text) &&"
```
Comment (zaro0508) [inline ec2/sc-ec2-linux-jumpcloud.yaml]: `&&` doesn't seem necessary here?

File: ec2/sc-ec2-linux-jumpcloud.yaml [hunk 2]
```diff
@@ -386,14 +386,15 @@ Resources:
             01_tag_root_disk:
               command: !Join
                 - ''
-                - - "EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) "
+                - - "set -x;"
+                  - "EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) &&"
```
Comment (zaro0508) [inline ec2/sc-ec2-linux-jumpcloud.yaml]: `&&` doesn't seem necessary here?  at least it worked as is here: https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/managed-ec2-linux-v1.j2#L220

File: ec2/sc-ec2-linux-jumpcloud.yaml [hunk 3]
```diff
@@ -386,14 +386,15 @@ Resources:
...
                   - "ROOT_DISK_ID=$(/usr/bin/aws ec2 describe-volumes --region $AWS_REGION "
                   - "--filters Name=attachment.instance-id,Values=$EC2_INSTANCE_ID "
-                  - "--query Volumes[].VolumeId --out text); "
-                  - "EC2_INSTANCE_TAGS=$(aws --region $AWS_REGION ec2 describe-tags --filters 'Name=resource-id,Values=$EC2_INSTANCE_ID') "
-                  - "PROJECT=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"Project\").Value ') "
-                  - "DEPARTMENT=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"Department\").Value ') "
-                  - "OWNER_EMAIL=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"OwnerEmail\").Value ') "
+                  - "--query Volumes[].VolumeId --out text) &&"
+                  - "EC2_INSTANCE_TAGS=$(aws --region $AWS_REGION ec2 describe-tags --filters Name=resource-id,Values=$EC2_INSTANCE_ID) &&"
```
Comment (zaro0508) [inline ec2/sc-ec2-linux-jumpcloud.yaml]: you may need a space after every `&&`?

File: ec2/sc-ec2-linux-jumpcloud.yaml (hunk 1 above)
Comment (tthyer) [reply ec2/sc-ec2-linux-jumpcloud.yaml]: All changes made have been tested and were necessary

File: ec2/sc-ec2-linux-jumpcloud.yaml (hunk 2 above)
Comment (tthyer) [reply ec2/sc-ec2-linux-jumpcloud.yaml]: This script is working as is

Comment (tthyer) [reply ec2/sc-ec2-linux-jumpcloud.yaml]: All changes made have been tested and were necessary

File: ec2/sc-ec2-linux-jumpcloud.yaml (hunk 3 above)
Comment (tthyer) [reply ec2/sc-ec2-linux-jumpcloud.yaml]: Spaces added

File: ec2/sc-ec2-linux-jumpcloud.yaml [hunk 4]
```diff
@@ -386,14 +386,15 @@ Resources:
...
+                  - "EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id) && "
                   - "ROOT_DISK_ID=$(/usr/bin/aws ec2 describe-volumes --region $AWS_REGION "
                   - "--filters Name=attachment.instance-id,Values=$EC2_INSTANCE_ID "
-                  - "--query Volumes[].VolumeId --out text); "
-                  - "EC2_INSTANCE_TAGS=$(aws --region $AWS_REGION ec2 describe-tags --filters 'Name=resource-id,Values=$EC2_INSTANCE_ID') "
-                  - "PROJECT=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"Project\").Value ') "
-                  - "DEPARTMENT=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"Department\").Value ') "
-                  - "OWNER_EMAIL=$(echo $EC2_INSTANCE_TAGS | jq '.Tags[] | select(.Key == \"OwnerEmail\").Value ') "
+                  - "--query Volumes[].VolumeId --out text) && "
+                  - "EC2_INSTANCE_TAGS=$(aws --region $AWS_REGION ec2 describe-tags --filters Name=resource-id,Values=$EC2_INSTANCE_ID) & &"
```
Comment (zaro0508) [inline ec2/sc-ec2-linux-jumpcloud.yaml]: woops?  `& &"`?

Comment (tthyer) [reply ec2/sc-ec2-linux-jumpcloud.yaml]: thanks fixed

**Pull request merged**